README.md

```diff
@@ -144,6 +144,8 @@ interface GitHub {
   @GET @Path("/repos/{owner}/{repo}/contributors")
   List<Contributor> contributors(@PathParam("owner") String owner, @PathParam("repo") String repo);
 }
+...
```

Comment:
Last nit: I would still retain newlines after the call to each builder. Some mobile viewers will see an awkward wrap otherwise.